openmetadata-docs/content/developers/contribute/build-code-and-run-tests/openmetadata-server.md

```diff
@@ -11,7 +11,13 @@ Learn how to run the OpenMetadata server in development mode by using Docker and
   - For an easy install of MySQL and ES, just install Docker on your local machine and run the following commands from the top-level directory
 
 ```shell
-docker-compose -f docker/local-metadata/docker-compose-dev.yml up
+docker-compose -f docker/local-metadata/docker-compose.yml -d up mysql elasticsearch
+```
+
+- For an easy install of PostgreSQL and ES, just install Docker on your local machine and run the following commands from the top-level directory
+
+```shell
+docker-compose -f docker/local-metadata/docker-compose-postgres.yml -d up postgresql elasticsearch
```

Comment:
Let's update the command to use `docker compose` as per the Issue Task [here](https://github.com/open-metadata/OpenMetadata/issues/7099#issuecomment-1272788163)

Reply:
Hey @akash-jain-10 , I have made the changes as per task. Can you pinpoint the changes I need to make here?

Reply:
Please note of the command mentioned. Instead of `docker-compose` we need `docker compose`